jest.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  preset: 'jest-preset-angular',
+  setupFilesAfterEnv: ['<rootDir>/setup-jest.ts'],
+  globals: {
+    "ts-jest": {
+      "tsconfig": "<rootDir>/projects/form-gen/core/tsconfig.spec.json"
```

Comment:
I am not quite happy with this line.

It's better to put `tsconfig.spec.json` into the root folder. If necessary, we also might put `tsconfig.spec.json` into each project folder and extend the root `tsconfig.spec.json`.

Reply:
I see. Will change it in a minute.